src/server/node_func.c

```diff
@@ -664,6 +664,7 @@ remove_mom_from_vnodes(mominfo_t *pmom)
 		}
 
 	}
+	free_arst(&tmomattr);
```

Comment:
Should this not be a call to .at_free? Maybe `node_attr_def[(int)ND_ATR_Mom].at_free(&tmomattr);`